sdk/src/main/java/com/podio/sdk/domain/field/Field.java

```diff
@@ -142,7 +143,8 @@ public boolean isVisible() {
         email(EmailField.class),
         phone(PhoneField.class),
         undefined(UndefinedField.class),
-        tag(OrganisationTagField.class);
+        tag(OrganisationTagField.class),
+        linkedaccountdata(LinkedAccountDataField.class);
```

Comment:
Good point, actually according to java naming conventions, enum values should be all caps and if it has two words they should be separated by an underscore. Keeping up with the convention that is being currently followed, adding "_" to separate the words.